plugins/wiki/app/views/pages/all.html.haml

```diff
@@ -6,7 +6,6 @@
 .navbar
   .navbar-inner
     %ul.nav
-      %li= link_to t('.recent_changes'), all_pages_path(:view => 'recent_changes')
```

Comment:
we do not miss this functionality! Now it's implemented via sort_helper like everywhere else. it's an usability improvement to make all pages behave in a similar way :-)